main.go

```diff
@@ -138,33 +132,40 @@ func caduceus(arguments []string) int {
 		modifiedWRPCount:         metricsRegistry.NewCounter(ModifiedWRPCounter),
 		maxOutstanding:           0,
 	}
+	measures := NewMeasures(metricsRegistry)
 
-	webhookFactory, err := webhook.NewFactory(v)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error creating new webhook factory: %s\n", err)
-		return 1
+	var updateListSizeMetric chrysom.ListenerFunc
+	updateListSizeMetric = func(items []model.Item) {
+		measures.WebhookListSize.Set(float64(len(items)))
 	}
-	webhookRegistry, webhookHandler := webhookFactory.NewRegistryAndHandler(metricsRegistry)
-	webhookFactory.SetExternalUpdate(caduceusSenderWrapper.Update)
+	webhookRegistry, err := NewRegistry(RegistryConfig{
+		Logger: logger,
+		Listener: func(items []model.Item) {
+			hooks := make([]webhook.W, len(items))
+			for index, item := range items {
+				hook, err := convertItemToWebhook(item)
+				if err != nil {
+					continue
```

Comment:
we should probably log this or put it in a metric or something - we don't expect any failures here and suppressing this error could make it more difficult to debug later.